domains/omarjaber.json

```
@@ -0,0 +1,11 @@
+{
+    
+    "repo": "https://github.com/omarHJ/Portfolio-Website",
+    "owner": {
+        "username": "omarHJ",
+        "email": "omarpp5555@gmail.com"
+    },
+    "record": {
+        "CNAME": "omarHJ.github.io"
```

404 site not found, if the records are correct then provide the website link
```suggestion
        "CNAME": "omarhj.github.io"
```

---

do you mean this: https://omarhj.github.io/Portfolio-Website/  ??